docs/api.yml

```diff
@@ -0,0 +1,226 @@
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queue/{queue_name}/empty:
+    post:
+      summary: Empty the queue with the specified queue_name
+      parameters:
+        - name: queue_name
+          in: path
+          schema:
+            type: string
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queue/{queue_name}/delete:
+    post:
+      summary: Delete the queue with the specified queue_name
+      parameters:
+        - name: queue_name
+          in: path
+          schema:
+            type: string
+          required: true
+      responses:
+        200:
+          description: Request processed
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+  /queues.json:
+    get:
+      summary: List Queues and their state
+      responses:
+        200:
+          description: |
+            Return object containting list of queues and theis states
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  queues:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        name:
+                          type: string
+                        count:
+                          type: integer
+                        url:
+                          type: string
+                          format: url
+  /jobs/{queue_name}/{page}.json:
+    get:
+      summary: List workers and their state
+      parameters:
+        - name: queue_name
+          in: path
+          schema:
+            type: string
+          required: true
+        - name: page
+          in: path
+          schema:
+            type: integer
+          required: true
+      responses:
+        200:
+          description: |
+            Return object containting list of worker states in workers property
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  name:
+                    type: string
+                  jobs:
+                    type: array
+                    items:
+                      type: object
+                      properties:
+                        id:
+                          type: string
+                          format: uuid
+                        status:
+                          type: string
+                        enqueued_at:
```

Comment:
`ended_at` is missing https://github.com/djangsters/rt-dashboard/blob/7008ffdb55db848f5882957de16725ba5748c279/rt_dashboard/web.py#L123

Reply:
for running jobs we also return `worker` https://github.com/djangsters/rt-dashboard/blob/7008ffdb55db848f5882957de16725ba5748c279/rt_dashboard/web.py#L281